### PR TITLE
Allow whitespace for company name

### PR DESCRIPTION
**Pull request by kgevorkyan**
### What's done:
* Allow whitespace for company name

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/validation/InputValidationUtils.kt [hunk 1]
```diff
@@ -7,6 +7,8 @@ package com.saveourtool.save.frontend.components.views.usersettings.right.valida
 import com.saveourtool.save.frontend.utils.UsefulUrls
 import com.saveourtool.save.validation.*
 
+private val namingAllowedSymbols = setOf('-', '_', '.', ' ')
```
[the remark this answers is not in the document]
Comment (kgevorkyan) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/validation/InputValidationUtils.kt:10]: its common `set` of allowed symbols

* for Name I guess we shouldnt have nor `,` neither `
* for company - will add `,` to allowed symbols

but about apostrophe -- dont know at all, do we really need?

[the remark this answers is not in the document]
Comment (nulls) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/validation/InputValidationUtils.kt:10]: I googled this one: https://www.wordstream.com/blog/ws/2021/11/08/business-name-ideas
- `Something, Inc.`
- `Someone's`

[the remark this answers is not in the document]
Comment (kgevorkyan) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/validation/InputValidationUtils.kt:10]: o, thought about ` and not '

though that you are talking about some French specific words or smth like this

will add `,` and ' to company name

[the remark this answers is not in the document]
Comment (kgevorkyan) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/validation/InputValidationUtils.kt:10]: changed

**Review by Cheshiriks — APPROVED** [review]

**Review by DrAlexD — APPROVED** [review]

**Pull request merged**